### ISSUE TITLE
Integrate with Godot

### ISSUE THREAD

**Issue by kvark** [gfx-rs/gfx#1791]
[Godot](https://godotengine.org/) is one of the most popular open source game engines, and it's currently using GL only. It would be great to have a tighter collaboration between our teams, and I could see the following paths:
  1. Provide them with our Vulkan [portability](https://github.com/gfx-rs/portability) implementation. This can be started right now, with our task to assist the porting efforts, quickly identify and fix upcoming issues.
  2. Rewrite the rendering component (following the [visual server](https://github.com/godotengine/godot/blob/34c988cfa92f19c232b65990704816ba1c7d2622/servers/visual/visual_server_raster.h) protocol, as far as I see) in Rust. It's supposed to run on a separate thread(s), and we can then call gfx-hal directly from it.

**Comment by karroffel** [comment]
Yes I am working on a rendering engine with C/C++ API, it can already present to the Godot-created window and right now I'm in the process of implementing a render graph implementation. I'll post updates later on!

**Comment by vultix** [comment]
@karroffel Thank you for your work on this! What's the current status of the project?

**Comment by karroffel** [comment]
@vultix unfortunately I am no longer working on that project.
